<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Run on Chrome"
           tests="3"
           failures="1"
           errors="0"
           skipped="0"
           time="31.570">
  <properties />
            <testcase name="TC_01_CreateNewCustomerAccountAndGetID" time="17.497" classname="com.guru.testcases.LoginTest">
                <failure type="java.lang.AssertionError"
                  message="Layout test failed"
              >
<![CDATA[java.lang.AssertionError: Layout test failed
  org.testng.Assert.fail(Assert.java:94)
  common.AbstractPage.checkUIpage(AbstractPage.java:82)
  demoGuruPages.HomePage.checkUI(HomePage.java:22)
  com.guru.testcases.LoginTest.TC_01_CreateNewCustomerAccountAndGetID(LoginTest.java:88)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
  org.testng.SuiteRunner.run(SuiteRunner.java:240)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
  org.testng.TestNG.run(TestNG.java:1031)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:282)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:83)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:114)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:286)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:240)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="TC_04_TransferMoneyIntoCurrentAccount" time="8.167" classname="com.guru.testcases.LoginTest" />
                <testcase name="TC_03_AddNewAccountAndGetAccountID" time="5.906" classname="com.guru.testcases.LoginTest" />
      
</testsuite>
